src/components/Map.tsx

```diff
@@ -33,7 +33,14 @@ const Map = ({
     >
       <MapEvents />
       <Marker position={marker}>
-        <Popup>Upload an image to this location!</Popup>
+        <Popup>
+          <button
+            className="flex-shrink-0 bg-blue-500 hover:bg-blue-700 border-blue-500 hover:border-blue-700 text-md border-4 text-white py-1 px-4 rounded"
```

Comment:
I think this button would be a good candidate to test extracting tailwind CSS components so we don't have to repeat the same classes to achieve the same button elsewhere.

- https://tailwindcss.com/docs/extracting-components
- https://www.youtube.com/watch?v=v-mkUxhaFVA

Reply:
We can also maybe do the same for the login buttons

Reply:
I agree. I don't like any part of the UI at the moment, and, yeah, the inline CSS is already starting to become spaghetti. I can take a look at it after I finish authentication.